components/Benefit/Benefit.tsx

```diff
@@ -0,0 +1,17 @@
+import type { NextPage } from "next";
+import styles from "styles/Benefit.module.scss";
+
+type BenefitProps = {
+    text: string;
+};
+
+const Benefit: NextPage<BenefitProps> = ({ text }) => {
```

Comment:
We likely don’t want `NextPage` on there since this isn’t a page.

```suggestion
const Benefit: React.FC<BenefitProps> = ({ text }) => {
```